data/mods/CrazyCataclysm/speech.json

```diff
@@ -250,5 +250,253 @@
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Then who the hell else are you talkin' to?\"",
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Well I'm the only one here.\"",
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Who the fuck do you think you're talking to?\"",
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"What we've got here is failure to communicate.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"E.T. phone home.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"I'm as mad as hell, and I'm not going to take this anymore!\"",
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"There's no place like home.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Badges?  We ain't got no badges!\"",
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"We don't need no badges!  I don't have to show you any stinking badges!\"",
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"I'll be back.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"I see dead people.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"It's alive! It's alive!\"",
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Say hello to my little friend!\"",
+    "volume" : 60
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Here's Johnny!\"",
+    "volume" : 30
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Open the pod bay doors, HAL.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"They may take our lives, but they'll never take our freedom!\"",
+    "volume" : 40
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"If you let my daughter go now, that'll be the end of it.  I will not look for you, I will not pursue you.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"If you don't, I will look for you, I will find you, and I will kill you.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Roads? Where we're going we don't need roads.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Fasten your seatbelts. It's going to be a bumpy night.\"",
+    "volume" : 20
+  },
+  {  },
```

Comment:
Extra brackets

Reply:
Oh, so that's what why Jenkins was so annoyed, I thought I got them all. Thanks